supportApp/videoCompressionSrc/codec_tools.c

```diff
@@ -0,0 +1,222 @@
+/* Streaming video compression tools for libavcodec */
+/* Author: B.A. Sobhani <sobhaniba@ornl.gov> */
+
+#include <libavcodec/avcodec.h>
+#include <libavutil/imgutils.h>
+#include <libavutil/opt.h>
+#include "streaming_vc.h"
+//#include "pgm.h"
+#include "codec_tools.h"
+
+AVCodecContext *c_c = 0; 
+int first = 1;
+AVCodecContext *c_d = 0; 
+int count_d = 0;
+
+
+const AVCodec *codec;
```

Comment:
Wouldn't it be best to avoid global variables? Doesn't this mean a single IOC can only have a single compressed stream?